dom-manipulation/dom-manipulation-quiz-notes.md

```diff
@@ -0,0 +1,48 @@
+# dom-manipulation-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is the `className` property of element objects?
+  A: className property allows you to access and manipulate by CSS classes applied to an element.
+- How do you update the CSS class attribute of an element using JavaScript?
+  A: with the getter and setter functionalities. query select and assign to variable then use className property of variable to update.
+- What is the `textContent` property of element objects?
+  A: textContent represents an elements text content and all its descendants.
+- How do you update the text within an element using JavaScript?
+  A: getter/setter, similar to using className
```

Comment:
Answer we're looking for here: modifying the `textContent` property of the element.
getter/setter refers to the attributes of an element, such as 'id' or 'class'.